openmetadata-spec/src/main/resources/json/schema/entity/services/metadataService.json

```diff
@@ -3,12 +3,18 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Metadata Service",
   "description": "This schema defines the Metadata Service entity, such as Amundsen, Atlas etc.",
+  "javaType": "org.openmetadata.schema.entity.services.MetadataService",
+  "javaInterfaces": [
+    "org.openmetadata.schema.EntityInterface",
+    "org.openmetadata.schema.ServiceEntityInterface"
+  ],
   "type": "object",
   "definitions": {
     "metadataServiceType": {
       "description": "Type of database service such as Amundsen, Atlas...",
+      "javaInterfaces": ["org.openmetadata.schema.EnumInterface"],
       "type": "string",
-      "enum": ["Amundsen", "MetadataES", "OpenMetadata", "Atlas"],
+      "enum": ["Amundsen", "MetadataES", "OpenMetadata", "Atlas", "MetadataToElasticSearch"],
```

Comment:
we don't want this "MetadataToElasticSearch" as a service right. It should be a pipeline type